### PR TITLE
Kata deploy 1.2 dockerfile

### PR DESCRIPTION
**Pull request by egernst**
update dockerfile to 1.2.0

Update yaml to use explicit versions, since 1.2.0 is introducing changes in how the tarball is formatted.

### CODE REVIEW

File: kata-deploy/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,14 @@
 FROM centos/systemd
-ARG KATA_VER=1.1.0
+ARG KATA_VER=1.2.0
 ARG KATA_URL=https://github.com/kata-containers/runtime/releases/download/${KATA_VER}
+ARG KATA_FILE=kata-static-${KATA_VER}-x86_64.tar.xz
```
Comment (jodh-intel) [inline kata-deploy/Dockerfile]: I wonder if we want to put that architecture in a variable (or rename the file to `Dockerfile.x86_64`)?

**Comment by jodh-intel** [comment]
lgtm

File: kata-deploy/Dockerfile (hunk 1 above)
Comment (egernst) [reply kata-deploy/Dockerfile]: I'll set it to a variable

File: kata-deploy/kata-cleanup.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ spec:
           kata-containers.io/kata-runtime: cleanup
       containers:
       - name: kube-kata-cleanup
-        image: katadocker/kata-deploy
+        image: katadocker/kata-deploy:1.1.0
```
Comment (jcvenegas) [inline kata-deploy/kata-cleanup.yaml]: 1.1.0 or 1.2.0 ?

Comment (jcvenegas) [reply kata-deploy/kata-cleanup.yaml]: same for the rest of  references.

Comment (egernst) [reply kata-deploy/kata-cleanup.yaml]: 1.1.0.  The format changed in 1.2.0, so we need existing dockerfile for the yaml that are in repo today.  Once we have this PR landed (ie, a 1.2.0 dockerfile available), we'll update the yamls to make use of this (at which point latest should be fine again).

**Comment by jcvenegas** [comment]
lgtm

**Pull request merged**